packages/conf-mingw-w64-ao-x86_64/conf-mingw-w64-ao-x86_64.1/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "2.0"
+synopsis: "libao for x86_64 mingw-w64 (64-bit x86_64)"
+description: "Ensures the x86_64 version of libao for the mingw-w64 project is available"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+authors: "libao dev team"
+license: "GPL-2.0-only"
+homepage: "https://www.xiph.org/ao/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+flags: conf
+available: os = "win32"
+build: ["pkgconf" "--personality=x86_64-w64-mingw32" "ao"]
+depends: [
+  "conf-pkg-config" {build}
+  "conf-mingw-w64-gcc-x86_64" {build}
+]
+depexts: [
+  ["mingw64-x86_64-libao"] {os = "win32" & os-distribution = "cygwin"}
```

Comment:
While working on my [blog post about libao](https://www.gridbugs.org/sound-on-ocaml-on-windows/) I used exactly this depext entry for cygwin and found that it didn't work properly. The `mingw64-x86_64-*` package seem to install to <cywgin>\root\usr\x86_64-w64-mingw32\sys-root\mingw\lib, and I found that pkgconf doesn't look in this directory when searching for libraries. I only managed to get this working on msys2 (using the exact same line you have below).

Reply:
Ohh interesting, no I did not. That might be what I was doing wrong. Let me see if that works for me!